tests/unit/gensym_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest 
+
+from lvm_gensym import * 
+
+used_vg_names = ['rhel_tflannag_bos_csb', 'rhel_tflannag_bos_csb_0', 'rhel_tflannag_bos_csb_1']
+used_lv_names = ['root', 'root_0', 'root_1', 'root_2', 'swap_0', 'swap', 'swap_1']
+test_input = [ ['ext', '/'], ['swap', '/'], ['ext', '/'], ['ext', '']]
```

Comment:
The second one shouldn't have any mountpoint -- swap doesn't get mounted like file systems do.